spec/features/stock_counter_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "StockCounter", type: :feature, js: true do
```

Comment:
Sorry about this. I changed the class name to "Stock" but forgot to change the test description in the spec file. I will fix this immediately.